src/utils/api/member.ts

```diff
@@ -158,3 +160,17 @@ export async function completeUpdatePaymentSource(
   );
   return toContrib(data);
 }
+
+export async function fetchPayments(
```

Comment:
I've been wondering about renaming this to `fetchContributionHistory`.... but that wouldn't be accurate, would it

Reply:
I can see the logic to this but I think it would need to be part of a wider renaming job so will leave as `fetchPayments` for now